### PR TITLE
Create chain-monitoring.mdx

### PR DESCRIPTION
**Pull request by cpengilly**

### CODE REVIEW

**Comment by sbvegan** [comment]
Here's some helpful context on dispute-mon


- Other Useful Notes
    - `op-dispute-mon` is set up to be built and run the same way as the `op-challenger`. Just `make op-dispute-mon` in its directory.
    - Since `op-dispute-mon` inherits the same metrics + logging configuration, the config from above can be used with an `OP_DISPUTE_MON` prefix.
    - A potential configuration for this chain could look like the following
        
        ```bash
        OP_DISPUTE_MON_LOG_FORMAT=logfmt
        OP_DISPUTE_MON_METRICS_ENABLED=true
        OP_DISPUTE_MON_METRICS_ADDR=0.0.0.0
        OP_DISPUTE_MON_METRICS_PORT=7300
        
        OP_DISPUTE_MON_L1_ETH_RPC=..
        OP_DISPUTE_MON_ROLLUP_RPC=..
        OP_DISPUTE_MON_GAME_FACTORY_ADDRESS=..
        
        OP_DISPUTE_MON_HONEST_ACTORS=..
        ```
        
        > Where `OP_DISPUTE_MON_HONEST_ACTORS` is a CSV (no spacing) list of addresses that are used for the honest `op-challenger` instance(s).
        > 
        
        Note, you may also ignore specific game instances with `OP_DISPUTE_MON_IGNORED_GAMES`, though you shouldn’t have to do this. 
        
    - Additional Flags
        - `OP_DISPUTE_MON_GAME_WINDOW`: This is the window of time to report on games. It should leave a buffer beyond the max game duration for bond claiming. If Fault Proof game parameters are not changes (e.g. `MAX_CLOCK_DURATION`), it is recommended to leave this as the default.
        - `OP_DISPUTE_MON_MONITOR_INTERVAL`: The interval at which to check for new games. Defaults to 30 seconds currently.
        - `OP_DISPUTE_MON_MAX_CONCURRENCY`: The max thread count. Defaults to 5 currently.

**Comment by sbvegan** [comment]
This PR addresses: https://github.com/ethereum-optimism/docs/issues/345

File: pages/builders/chain-operators/tools/chain-monitoring.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+## Onchain Monitoring Services
+
+Onchain monitoring services provide insights into the overall system and helps chain operators track and monitor onchain events. Some examples of onchain monitoring services include `monitorism` and `dispute-mon`.
+
+### `monitorism`
+
+Monitorism is a tooling suite that supports monitoring and active remediation actions for the OP Stack chain. Monitorism uses monitors as passive security providing automated monitoring for the OP Stack. They are used to monitor the OP stack and alert on specific events that could be a sign of a security incident.
+
+Currently. the list of monitors includes:
```
Comment (raffaele-oplabs) [inline pages/builders/chain-operators/tools/chain-monitoring.mdx:22]: I would divide this list in two parts:
1. Security Integrity Monitor: These are monitors necessary for making sure Bridges between L2 and L1 are safe and work as expected. These monitors are divided in two subgroup:
1. a Pre-Faultproof chain monitors: Fault Monitor, Withdrawals Monitor
2.b Faultproof chain monitors: Faulproof Withdrawals
2. Security monitors: Those tools monitor other aspects of several contracts used in optimism.
Global Events Monitor,Liveness Expiration Monitor,Balances Monitor,Multisig Monitor,Drippie Monitor,Secrets Monitor

Comment (krofax) [reply pages/builders/chain-operators/tools/chain-monitoring.mdx:22]: @bradleycamacho

**Comment by bradleycamacho** [comment]
@krofax mind giving this a review?

File: pages/builders/chain-operators/tools/chain-monitoring.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+Security monitors: Those tools monitor other aspects of several contracts used in optimism:
+
+*   Global Events Monitor: made for to taking YAML rules as configuration and monitoring the events that are emitted on the chain.
+*   Liveness Expiration Monitor: monitors the liveness expiration on Safes.
+*   Balances Monitor: emits a metric reporting the balances for the configured accounts.
+*   Multisig Monitor: The multisig monitor reports the paused status of the OptimismPortal contract. If set, the latest nonce of the configured Safe address. And also if set, the latest presigned nonce stored in One Password. The latest presigned nonce is identified by looking for items in the configured vault that follow a `ready-<nonce>.json` name. The highest nonce of this item name format is reported.
```
[the remark this answers is not in the document]
Comment (krofax) [reply pages/builders/chain-operators/tools/chain-monitoring.mdx:37]: This sentence - "If set, the latest nonce of the configured Safe address. And also if set, the latest presigned nonce stored in One Password." sounds confusing.

Do we want to reword it to -> "If set, reports the latest nonce of the configured Safe address and the latest presigned nonce stored in One Password."? @bradleycamacho 

Combining the two sentences for clarity

File: pages/builders/chain-operators/tools/chain-monitoring.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+*   Drippie Monitor: tracks the execution and executability of drips within a Drippie contract.
+*   Secrets Monitor: takes a Drippie contract as a parameter and monitors for any drips within that contract that use the `CheckSecrets` dripcheck contract. `CheckSecrets` is a dripcheck that allows a drip to begin once a specific secret has been revealed (after a delay period) and cancels the drip if a second secret is revealed. It's important to monitor for these secrets being revealed as this could be a sign that the secret storage platform has been compromised and someone is attempting to exfiltrate the ETH controlled by that drip.
+
+For more information on these monitors and how to use them, [check out the repo](https://github.com/ethereum-optimism/monitorism?tab=readme-ov-file#monitorism).
+
+### `dispute-mon`
+
+Chain operators should consider running `op-dispute-mon`. It's an incredibly useful securities monitoring service to keep an eye on games, basically giving visibility into all the status of the games for the last 28 days.
```
Comment (krofax) [inline pages/builders/chain-operators/tools/chain-monitoring.mdx:45]:
```suggestion
Chain operators should consider running `op-dispute-mon`. It’s an essential security monitoring service that tracks game statuses, providing visibility over the last 28 days.
```

Reworded for a more formal tone.

File: pages/builders/chain-operators/tools/chain-monitoring.mdx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    --metrics.enabled                   (default: false)                   ($OP_CHALLENGER_METRICS_ENABLED)
+          Enable the metrics server
+   
+    --metrics.port value                (default: 7300)                    ($OP_CHALLENGER_METRICS_PORT)
+          Metrics listening port
+```
+
+## Next Steps
+
+*   If you experience difficulty at any stage of this process, please reach out to [developer support](https://github.com/ethereum-optimism/developers/discussions).
```
Comment (krofax) [inline pages/builders/chain-operators/tools/chain-monitoring.mdx:128]:
```suggestion
*   If you encounter difficulties at any stage of this process, please reach out to [developer support](https://github.com/ethereum-optimism/developers/discussions).
```

**Comment by krofax** [comment]
LGTM

**Review by krofax — APPROVED** [review]
